app/controllers/public/index.js

```diff
@@ -2,6 +2,7 @@ import classic from 'ember-classic-decorator';
 import { action, computed } from '@ember/object';
 import Controller from '@ember/controller';
 import { htmlSafe } from '@ember/string';
+import { allSettled } from 'rsvp';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['allSettled' is defined but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5583663)